README.md

```diff
@@ -31,6 +31,7 @@ virtualenv must be installed on the system.
 - monasca_user_domain_name: Keystone project domain name for Monasca
 - monasca_user_domain_id: Keystone project domain ID for Monasca
 - monasca_user: User which runs the Monasca Agent. Defaults to 'mon-agent'.
+- monasca_agent_custom_plugin_repos: A list of git repos containing custom plugins.
```

Comment:
Could we specify a branch here?

Reply:
That's possible. For example:

```
monasca_agent_custom_plugin_repos:
      - "https://github.com/stackhpc/stackhpc-monasca-agent plugins.git@feature/add_ib_detection_plugin" 
```

Reply:
Ok, perhaps a comment to specify what the format is?